terminus-ui/src/file-upload/selected-file.ts

```diff
@@ -211,4 +217,19 @@ export class TsSelectedFile {
     return false;
   }
 
+  private validateImageRatio(constraints: Array<number> | undefined): boolean {
```

Comment:
I'm afraid we are going to have to make this a bit more complex.. Ratio's are most commonly defined as two numbers separated by a `:` eg `3:4`. So I think we'll need to support an array of strings rather than numbers, then use the `:` to split the string.

Which means we'll have to add some logic to convert the image dimensions into a ratio. I'm guessing we should probably round the ratios also. It looks like LinkedIn needs 2 decimal places so we can just go with that for now.